hawkpost/templates/account/auth_modals.html

```diff
@@ -35,29 +32,27 @@ <h3 class="title "> Login</h3>
               </p>
             </label>
             <label class="input-anim" for="id_remember">
-              <p> {{ login_form.remember.errors }}
-                <div class="row">
-                    <div class="col-xs-6">
-                        <div class="row">
-                            <div class="col-xs-8">
-                                {{ login_form.remember.label_tag }}
-                            </div>
-                            <div class="col-xs-2 end-xs">
-                                {{ login_form.remember }}
-                            </div>
-                        </div>
-                    </div>
+              <p class="no-padding"> {{ login_form.remember.errors }}
+                <div class="flex-in">
+                  {{ login_form.remember.label_tag }}
+                  {{ login_form.remember }}
                 </div>
               </p>
             </label>
           </div>
           {% if redirect_field_value %}
-            <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
+          <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
           {% endif %}
         </div>
         <div class="col-xs-12">
           <button class="primaryAction btn-default md-trigger" data-modal="modal-login" type="submit">{% trans "Login" %}</button>
         </div>
+        <div class="col-xs-12 middle-xs flex-in mt-small">
+          <p class="no-padding">{% blocktrans with site.name as site_name %}Login in via:  {% endblocktrans %}</p>
+          <ul class="gitpad no-margin">
+            {% include "socialaccount/snippets/provider_list.html" with process="login" %}
```

Comment:
I think this include is dependent on the conditional of line 14 (`{% if socialaccount_providers %}`), no? probably you should also check it here.